### PR TITLE
SAK-46974: Messages. Provide private message scheduling

### PR DESCRIPTION
**Pull request by victorGomollon**
[SAK-46974: Messages. Provide private message scheduling](https://sakaiproject.atlassian.net/browse/SAK-46974)
AND
[SAK-47389: MESSAGES: Scheduling - Avoid DEFAULT NULL](https://sakaiproject.atlassian.net/browse/SAK-47389)

### CODE REVIEW

File: docs/conversion/sakai_23_mysql_conversion.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+-- SAK-46974
+ALTER TABLE MFR_MESSAGE_T ADD COLUMN SCHEDULER bit(1) DEFAULT NULL;
```
Comment (bjones86) [inline docs/conversion/sakai_23_mysql_conversion.sql:2]: I would prefer to see this column not nullable, and with a default.

**Review by bjones86 — CHANGES_REQUESTED** [review]

File: docs/conversion/sakai_23_oracle_conversion.sql [hunk 2]
```diff
@@ -0,0 +1,4 @@
+-- SAK-46974
+ALTER TABLE MFR_MESSAGE_T ADD COLUMN SCHEDULER NUMBER(1,0) DEFAULT NULL;
```
Comment (jesusmmp) [inline docs/conversion/sakai_23_oracle_conversion.sql:2]: ALTER TABLE MFR_MESSAGE_T ADD SCHEDULER NUMBER(1,0) DEFAULT NULL;

File: docs/conversion/sakai_23_oracle_conversion.sql [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+ALTER TABLE MFR_MESSAGE_T ADD COLUMN SCHEDULED_DATE TIMESTAMP DEFAULT NULL;
```
Comment (jesusmmp) [inline docs/conversion/sakai_23_oracle_conversion.sql:3]: ALTER TABLE MFR_MESSAGE_T ADD SCHEDULED_DATE TIMESTAMP DEFAULT NULL;

File: docs/conversion/sakai_23_oracle_conversion.sql (hunk 2 above)
Comment (bjones86) [reply docs/conversion/sakai_23_oracle_conversion.sql:2]: Is this a boolean value? If so, why null for the default? Select a more appropriate default, either on or off depending on the intention of the feature.

File: docs/conversion/sakai_23_mysql_conversion.sql (hunk 1 above)
Comment (victorGomollon) [reply docs/conversion/sakai_23_mysql_conversion.sql]: Done

File: docs/conversion/sakai_23_oracle_conversion.sql (hunk 2 above)
Comment (victorGomollon) [reply docs/conversion/sakai_23_oracle_conversion.sql]: Done

File: docs/conversion/sakai_23_oracle_conversion.sql (hunk 3 above)
Comment (victorGomollon) [reply docs/conversion/sakai_23_oracle_conversion.sql]: Done

**Pull request merged**